src/api/response-types/legacy-namespace.ts

```diff
@@ -1,7 +1,7 @@
 export class LegacyNamespaceListType {
   id: number;
   url: string;
-  summary_fields: unknown;
+  summary_fields: any;
```

Comment:
```suggestion
  summary_fields: {
    owners?: { username: string; }[];
  };
```

should match the type it expects :)

Reply:
We can do this.